### PR TITLE
182089941 log string fixes

### PR DESCRIPTION
**Pull request by eireland**
Adds missing log messages
Replaces some logs to use log utility functions

### CODE REVIEW

File: v3/src/components/graph/components/parent-toggles.tsx [hunk 1]
```diff
@@ -178,7 +180,8 @@ export const ParentToggles = observer(function ParentToggles() {
       },
       {
         undoStringKey: "DG.mainPage.mainPane.undoButton.toolTip",
-        redoStringKey: "DG.mainPage.mainPane.redoButton.toolTip"
+        redoStringKey: "DG.mainPage.mainPane.redoButton.toolTip",
+        log: isOnlyLastShown ? "Disable last parent toggle" : "Enable last parent toggle"
```
Comment (kswenson) [inline v3/src/components/graph/components/parent-toggles.tsx:184]: Should probably be something more like:
```suggestion
        log: isOnlyLastShown ? "Disable only showing last parent toggle" : "Enable only showing last parent toggle"
```

File: v3/src/components/tool-shelf/options-button.tsx [hunk 2]
```diff
@@ -28,7 +29,8 @@ export const OptionsShelfButton = () => {
       isWebViewModel(tile?.content) && tile?.content.setUrl(url)
     }, {
       undoStringKey: "V3.Undo.webView.show",
-      redoStringKey: "V3.Redo.webView.show"
+      redoStringKey: "V3.Redo.webView.show",
+      log: logStringifiedObjectMessage("Show web view: %@", {url})
```
Comment (kswenson) [inline v3/src/components/tool-shelf/options-button.tsx:33]: Could use `logMessageWithReplacement` to just render the url without the braces.

File: v3/src/components/web-view/web-view-inspector.tsx [hunk 3]
```diff
@@ -30,7 +31,8 @@ export const WebViewInspector = observer(function WebViewInspector({tile, show}:
       webViewModel?.setUrl(url)
     }, {
       undoStringKey: "V3.Undo.webView.changeUrl",
-      redoStringKey: "V3.Redo.webView.changeUrl"
+      redoStringKey: "V3.Redo.webView.changeUrl",
+      log: logStringifiedObjectMessage("Change web view URL: %@", {url})
```
Comment (kswenson) [inline v3/src/components/web-view/web-view-inspector.tsx:35]: Could use `logMessageWithReplacement` to just render the url without the braces.

**Review by kswenson — CHANGES_REQUESTED** [review]
👍 Generally looks good -- a handful of minor suggestions.

File: v3/src/components/case-table/case-table-tool-shelf-button.tsx [hunk 4]
```diff
@@ -55,7 +55,7 @@ export const CaseTableToolShelfMenuList = observer(function CaseTableToolShelfMe
       notify: dataContextCountChangedNotification,
       undoStringKey: "V3.Undo.caseTable.create",
       redoStringKey: "V3.Redo.caseTable.create",
-      log: "createNewEmptyDataSet"
+      log: "create New Empty DataSet"
```
Comment (tealefristoe) [inline v3/src/components/case-table/case-table-tool-shelf-button.tsx:58]: Capitalize "Create"?

**Review by tealefristoe — APPROVED** [review]
Looks good 👍

File: v3/src/components/calculator/calculator-title-bar.tsx [hunk 5]
```diff
@@ -13,7 +14,8 @@ export const CalculatorTitleBar =
         documentContent?.toggleSingletonTileVisibility(kCalculatorTileType)
       }, {
         undoStringKey: "DG.Undo.toggleComponent.delete.calcView",
-        redoStringKey: "DG.Redo.toggleComponent.delete.calcView"
+        redoStringKey: "DG.Redo.toggleComponent.delete.calcView",
+        log: logStringifiedObjectMessage("Close calculator", { type: kCalculatorTileType })
```
[the remark this answers is not in the document]
Comment (eireland) [reply v3/src/components/calculator/calculator-title-bar.tsx:18]: Why add the calculator type to the log message string if we already state that we closed a calculator?

[the remark this answers is not in the document]
Comment (kswenson) [reply v3/src/components/calculator/calculator-title-bar.tsx:18]: Fair enough.

**Review by kswenson — APPROVED** [review]
👍 LGTM

**Pull request merged**